Examples/Framework/include/ActsExamples/Framework/SequenceElement.hpp

```diff
@@ -25,9 +25,12 @@ class SequenceElement {
  public:
   virtual ~SequenceElement() = default;
 
-  /// The algorithm name.
+  /// The sequence element name.
   virtual std::string name() const = 0;
 
+  /// The sequence element type name, used for debug output
+  virtual std::string_view typeName() const = 0;
```

Comment:
could be enum + `toString` / `toStream`

Reply:
It could be yes. It's only used in printouts so I didn't see a need to.